app/controllers/concerns/idv/document_capture_concern.rb

```diff
@@ -50,6 +50,24 @@ def selfie_requirement_met?
         stored_result.selfie_check_performed?
     end
 
+    def redirect_to_correct_vendor(vendor, in_hybrid_mobile)
```

Comment:
```suggestion
    def redirect_to_correct_vendor(vendor, in_hybrid_mobile?)
```

Reply:
I don't think you can suffix `?` in an argument name like that, unfortunately.

Reply:
sounds like i'm mixing up languages 😓